qa/rpc-tests/sigma_remint.py

```diff
@@ -56,6 +56,12 @@ def run_test(self):
         assert len(sigma_mint) == 12, 'Should be 12 sigma mints after remint, but was: {}' \
                 .format(len(sigma_mint))
 
+        sigma_mints = set([mint['amount'] for mint in sigma_mint])
+        expected_mints = set([Decimal('25.00000000'), Decimal('10.00000000'), Decimal('100.00000000'), Decimal('1.00000000')])
```

Comment:
why use set instead of list?

Reply:
Cause we checked the len before. Here just quick check that all denoms shown

Reply:
ok